jcl/src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -306,6 +309,29 @@ public void forEach(Consumer<? super StackFrame> action) {
 		 * @return StackTraceElement
 		 */
 		StackTraceElement toStackTraceElement();
+
+		/*[IF Java10]*/
+		/**
+		 * This method must be implemented in the implementing class.
+		 * @throws UnsupportedOperationException if this method is not overridden
```

Comment:
This should also mention that it will throw UOE unless the Stackwalker Options inclue RETAIN_CLASS_REFERENCE.

Reply:
Done.